pkg/agent/openflow/pipeline.go

```diff
@@ -2575,16 +2657,20 @@ func pipelineClassifyFlow(cookieID uint64, protocol binding.Protocol, pipeline b
 		Done()
 }
 
-// igmpPktInFlows generates the flow to load CustomReasonIGMPRegMark to mark the IGMP packet in MulticastRoutingTable and sends
-// it to antrea-agent.
+// igmpPktInFlows generates the flow to load CustomReasonIGMPRegMark to mark the IGMP packet in MulticastIGMPEgressTable
+// if multicast enabled or in MulticastRoutingTable if not and sends it to antrea-agent.
 func (f *featureMulticast) igmpPktInFlows(reason uint8) []binding.Flow {
+	table := MulticastIGMPEgressTable
+	if !f.enableAntreaPolicy {
+		table = MulticastRoutingTable
+	}
```

Comment:
I wonder if there is a point to move the packetin flow to MulticastIGMPEgressTable when policy is enabled, the flow is not only about egress policy anyway and is used for multiple purposes. Maybe we should just remove `MulticastIGMPEgressTable` and continue using MulticastRoutingTable for multiple purposes.

Reply:
the rule itself is not egress, but we handle egress in packetIn, do you think it ok to move the rule back to MulticastRoutingTable

Reply:
done